platform-hub-api/config/routes.rb

```diff
@@ -72,17 +74,19 @@
       post :unmark_sticky, on: :member
     end
 
-    namespace :kubernetes do
-      # Tokens management
-      get '/tokens/:user_id', to: 'tokens#index'
-      put '/tokens/:user_id/:cluster', to: 'tokens#create_or_update'
-      patch '/tokens/:user_id/:cluster', to: 'tokens#create_or_update'
-      delete '/tokens/:user_id/:cluster', to: 'tokens#destroy'
-      get '/clusters', to: 'clusters#index'
-      get '/changeset/:cluster', to: 'changeset#index'
-      post '/sync', to: 'sync#sync'
-      post '/claim', to: 'claim#claim'
-      post '/revoke', to: 'revoke#revoke'
+    constraints lambda { |request| FeatureFlagService.is_enabled?(:kubernetes_tokens) } do
```

Comment:
We should probably add a similar check in the `Identity` serialiser, so that the API doesn't return any token info if the feature is disabled (i.e. [here](https://github.com/UKHomeOffice/platform-hub/blob/master/platform-hub-api/app/serializers/identity_serializer.rb#L12)). That way we can also remove the logic in the UI, since if the API is not returning the data, then nothing is shown. (Though am happy for that logic to stay in, to be on the defensive side).